src/modules/sharing/sharing.service.ts

```diff
@@ -2057,4 +2058,14 @@ export class SharingService {
 
     return sharedItem;
   }
+
+  async getSharingFolderSize(id: SharingAttributes['id']): Promise<number> {
+    const sharing = await this.sharingRepository.findOneSharing({ id });
```

Comment:
Let's check if it is public. For public sharings, we basically add  sharedWith with the value  `00000000-0000-0000-0000-000000000000`, or so.

You can check in the endpoint GET sharing/meta how it works. So, check by id and sharedWith: `00000000-0000-0000-0000-000000000000`.  If there is no sharing with those conditions, then that sharing is not public, so you can return throw new SharingNotFoundException(); as you are doing right now.

Reply:
@sg-gs I made this review assuming we want this endpoint only for public sharings as it is public. If you want an endpoint that returns the size ignoring the sharing type, then please @edisonjpadilla, ignore the comment related the `sharedWith` field.